engine/src/flutter/shell/common/platform_view.h

```diff
@@ -121,6 +121,12 @@ class PlatformView {
     virtual void OnPlatformViewRemoveView(int64_t view_id,
                                           RemoveViewCallback callback) = 0;
 
+    /// @brief Notify the delegate that platform view focus state has changed.
+    ///
+    /// @param[in]  event  The focus event describing the change.
+    virtual void OnPlatformViewSendViewFocusEvent(
```

Comment:
Should we just default this OOL to be a no-op? You won't have to patch the mocks.

Reply:
No other delegate method has default implementation. If the inconsistency is not an issue we can do it.